### PR TITLE
Support array result include sequence action

### PR DESCRIPTION
**Pull request by ningyougang**
Depend on below prs: 
- https://github.com/apache/openwhisk/pull/5290  (merged)
- https://github.com/apache/openwhisk-runtime-go/pull/170 (merged)

- [x] Support array result
  - [x] make python runtime to support array result for common action
  - [x] make python runtime to support array result for sequence action (support array as input param)

### CODE REVIEW

**Comment by ningyougang** [comment]
Python supports sequence action by default with support array result, test steps as below
* Write split.py and sort.py
```py
[root@nccddev130026 ~]# cat ~/split.py 
def main(args):
    payload = args.get("payload", "aaaa\nbbbb\nccccc")
    separator = args.get("separator", "\n")
    result = payload.split(separator)
    return result

[root@nccddev130026 ~]# cat ~/sort.py 
def main(args):
    if args is None:
        args = ["data1", "data2", "data3"]
    result = args[::-1]
    return result
```

* Create action and invoke it (python:3.7/3.9 both tested, worked well both)
```shell
wsk -i action create /whisk.system/utils/split-python --kind python:3.7 ~/split.py
wsk -i action create /whisk.system/utils/sort-python --kind python:3.7 ~/sort.py
wsk -i action create mySequence-python --sequence /whisk.system/utils/split-python,/whisk.system/utils/sort-python
wsk -i action invoke --result mySequence-python --param payload "dddd\nbbbb\ncccc\nfffffff\nggggggggg" -r -v
```

File: core/python310Action/Dockerfile [hunk 1]
```diff
@@ -31,12 +31,12 @@ RUN curl -sL \
...
   | tar xzf -\
   && cd openwhisk-runtime-go-*/main\
-  && GO111MODULE=on go build -o /bin/proxy
+  && GO111MODULE=on CGO_ENABLED=0 go build -o /bin/proxy
 
 FROM python:3.10-buster
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=release
+ARG GO_PROXY_BUILD_FROM=source
```
Comment (ningyougang) [inline core/python310Action/Dockerfile:39]: Need to change it to `release` due to https://github.com/apache/openwhisk-runtime-java/pull/140#discussion_r939689436

Comment (ningyougang) [reply core/python310Action/Dockerfile]: Already changed to `release` to apply `1.18@1.20.0` tag

**Review by dgrove-oss — APPROVED** [review]
LGTM

**Pull request merged**